terraform/main.tf

```diff
@@ -25,18 +25,15 @@ resource "google_container_cluster" "primary" {
   initial_node_count = 2
 
   provisioner "local-exec" {
-    command = "gcloud container clusters get-credentials ${google_container_cluster.primary.name}"
+    command = "gcloud container clusters get-credentials ${google_container_cluster.primary.name} --zone ${google_container_cluster.primary.zone} --project ${var.project}"
   }
 
-  provisioner "local-exec" {
-    command  = "kubectl create configmap nginx-config --from-file=manifests/configs/default.conf"
-  }
+}
 
-  provisioner "local-exec" {
-    command  = "kubectl create configmap datadog-config --from-file=manifests/configs/conf.yaml"
-  }
+output "cluster_name" {
```

Comment:
https://github.com/GoogleCloudPlatform/gke-datadog-demo/blob/de426c7524ab1d23dd715ffaa27e940e2d15112b/terraform/main.tf#L33-L39

Do either of these outputs get used?

Reply:
As currently written the validate doesn't use these, though the other demos do so I left them here for commonality. I'd kind of like to keep them here because I think there's room to improve the validate. But I'm willing to remove them if you think it's necessary. I don't see harm in having them here.

Reply:
👍 Sounds good, leave it in